docs/getting-started/Quickstart.mdx

```diff
@@ -45,6 +45,14 @@ export default tseslint.config(
 
 This code will enable our [recommended configuration](../users/Shared_Configurations.mdx) for linting.
 
+:::note
+This file is in the [ES modules (ESM)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) format. Node interprets `.js` files in the [CommonJS (CJS)](https://nodejs.org/api/modules.html) format by default. To make the above not a syntax error, make sure to do one of the following:
+
+- Add `"type": "module"` to your `package.json`, so that all `.js` files are interpreted as ESM. (This may also affect other `.js` and `.ts` files in your project.)
+- Rename the file to `eslint.config.mjs`. This requires ESLint 8.57 or higher.
+- Refactor the config file to CJS instead, such as `const eslint = require('@eslint/js');` and `module.exports = tseslint.config(...);`.
+  :::
+
```

Comment:
This is a _lot_ of text to include in a quickstart guide. And it's on a topic many users don't care and/or are actively fearful about.

A couple of strategy suggestions:

* Instead of this `:::note`, maybe add `<Tab>`+`<TabItem>`s with names like `eslint.config.js (ESM)` and `eslint.config.js (CJS)`?
    * I considered suggesting titles like `eslint.config.(c)js` but that might just confuse users...
* Keep the note, but maybe link to a more direct guide (maybe one in eslint.org itself?)

IMO we should keep this page just to explaining getting started + a bit for next steps, not ancillary information covered elsewhere. My personal ideal would be to just add a tab for CJS config & leave it at that.